tools/ci_build/github/azure-pipelines/templates/stages/mac-ios-packaging-build-stage.yml

```diff
@@ -38,7 +38,7 @@ stages:
         cPodName: onnxruntime-training-c
         objcPodName: onnxruntime-training-objc
 
-    timeoutInMinutes: 180
+    timeoutInMinutes: 210
```

Comment:
note: I have a wip branch for splitting this ios packaging pipeline into per platform/per arch. after that, we can hopefully reduce the  pipeline building time.

so increase the CI timeout minutes for now.